src/views/graphql/journalEntryResolvers.ts

```diff
@@ -127,7 +127,7 @@ export class JournalEntryOverrideResolver {
             },
         });
 
-        if (existingJournalEntryDate)
+        if (existingJournalEntryDate && existingJournalEntryDate.id !== args.id)
```

Comment:
Checks for entry ids to match if there is one for the given date, if it does match then allow update but if it doesn't then throws error